.github/ISSUE_TEMPLATE/word-definition.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Word definition
+about: Template to define undefined word
+title: "➕ Propose definition for [word]"
+labels: Type · Writing, Type · Definitions
+assignees: ''
+
+---
+
+title: definition
+slug: slug
+speech: 
+alt_words:
+  - alt word
+  - alt word
+  - alt word
+reading:
+  - text: 'Link Title'
+    href: https://google.com
+  - text: 'Link Title'
+    href: https://google.com
+---
+
+Definition goes here. Below are some headings you can use in the definition. Generally 2-3 sentences max; include colloquial definitions as relevant.
+
+## Issues
+/* When the term is problematic, describe what issues there are. If term is not problematic, this can be omitted. */
+
+## Impact
+/* When the term is problematic, describe what impact there is. If term is not problematic, this can be omitted. */
+
+## Usage
+/* When there is nuance or considerations for a term's usage, describe that here. */
+
+## Benefits
+/* When term is a preferable alternative, describe the benefits */
+
+## Preferable To
+/* When term is a "Better Alternative", provide the inferior alternatives to avoid */
+
+## Considerations
+/* When a term needs nuance for its usage */
+
+## Appropriate Usage
+/* When term has appropriate and inappropriate usage; common for medical or culturally appropriative terms */
+
+## Read also
+/* When a term has related words that are defined */
+
+- [definition](/definitions/[word])
+- [definition](/definitions/[word])
+- [definition](/definitions/[word])
+
```

Comment:
Thinking about for the third time: I think we should move this into front matter and render this via a shortcode or loop, as we have the ability to automatically link words. That ability will be really helpful just in case the link structure changes. 

Plus, we could also link words that _will_ be defined, and the links will be added automatically. As a practical example, there’s this whole currently undefined complex of sexual health terms (PrEP, Undetectable, Viral Load …). By automating the linking we could define the list in all definitions and one by one they get linked without us having the edit n+1 files.

This also applies to #161